MyList.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'enumerable'
+
+class MyList
+  include Enumerable
+  def initialize(int1, int2, int3, int4)
+    @list = [int1, int2, int3, int4]
+  end
+end
```

Comment:
- [ ] great job on implementing the initialize method 👍  however according to the requirements MyList class should have the #initialize method that accepts *multiple arguments* not just 4. you can use the `*args` to specify arguments if you don't know the exact amount.

Reply:
Done, thank you for your feedback 👍🏼